internal/backends/qt/qt_widgets/listviewitem.rs

```diff
@@ -140,6 +147,11 @@ impl Item for NativeStandardListViewItem {
                     option.state |= QStyle::State_MouseOver;
                     option.state |= QStyle::State_Selected;
                 }
+
+                if (has_focus) {
+                    option.state |= QStyle::State_HasFocus;
+                    option.state |= QStyle::State_Selected;
```

Comment:
Is there a reason you added the selected state there?

Reply:
Yes I tried to get the focus visible. But this also dos not helped.